content/articles/gulp-sass-2.md

```diff
@@ -0,0 +1,124 @@
+Title: gulp-sass v2 released
+Date: 2015-05-06
+Short_summary: showing what's new in gulp-sass but most importantly in libsass
```

Comment:
This is what goes into a tweet so it should be a proper sentence with captialization etc.